packages/backend/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "baseUrl": "./src",
     "declaration": true,
     "esModuleInterop": true,
-    "lib": ["es2021"],
+    "lib": ["es2021", "ES2022.Error"],
```

Comment:
@ogp-weeloong @m0nggh this is necessary because we wanted to use the `cause` property for `Error` handling, right? shall we just upgrade to ES2022 directly as _i think_ they're backwards compatible? otherwise, the TypeScript APIs available will be a strange mix of both.

Reply:
hmm oh yea.. i was concerned whether it would be backwards compatible if we changed to es2022 entirely thats why i just added on first

Reply:
good thinking. I was discussing with @ogp-weeloong; I think we can port to es2022 in another subsequent PR so that we don't leave the codebase in a strange state between language API versions.